### DISCUSSION TITLE
buddhism/2024/eightfold_path/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#154] [General]
# buddhism/2024/eightfold_path/

https://shanenull.com/buddhism/2024/eightfold_path/

**Comment by shane0** [comment]
# eightfold path 

- samadhi https://shanenull.com/buddhism/2024/mind_ground/
- old version of this page https://shanenull.com/buddhism/2023/eightfold/

**Comment by shane0** [comment]
# Chapter 4 | The Ten Wholesome and Unwholesome Deeds

The Buddha said: In sentient beings, ten actions are wholesome and ten are unwholesome. What are they?

Three pertain to the body, four to the mouth, and three to the mind. Killing, stealing, and sexual misconduct pertain to the body. Malicious, abusive, false, and frivolous speech pertain to the mouth.

Envy, anger, and ignorance pertain to the mind. These actions are called the ten unwholesome deeds because they are not in accord with the noble way. To renounce the ten unwholesome actions is to practice the ten wholesome deeds.

**Comment by shane0** [comment]
# right view

- thich  https://youtu.be/JVLX8VtVU58
